### PR TITLE
WEBAPP-303: Add logo for sprungbrett-into-work

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

**Review by klinzo — APPROVED** [review]
Looking good :tada: Thanks!

Minor: Maybe it would look better if it was centered horizontally.

**Review by Taggotty — APPROVED** [review]

**Comment by Taggotty** [comment]
> Looking good 🎉 Thanks!
> 
> Minor: Maybe it would look better if it was centered horizontally.

I think so, too.
Still nice work :)

File: src/routes/sprungbrett/containers/SprungbrettOfferPage.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ export class SprungbrettOfferPage extends React.Component<PropsType> {
         <List noItemsMessage={t('noOffersAvailable')}
               renderItem={this.renderSprungbrettListItem}
               items={sprungbrettJobs} />
+        <a target='_blank' rel='noopener noreferrer' href='https://www.sprungbrett-intowork.de'>
```
Comment (maxammann) [inline src/routes/sprungbrett/containers/SprungbrettOfferPage.js:36]: I accidentally forgot to mention in the issue that the image should be centered horizontally. Ups, sorry!

This also fixes problems with RTL and LTR layouts. Centered is the same in both directions :)

Comment (maxammann) [inline src/routes/sprungbrett/containers/SprungbrettOfferPage.js:36]: :+1: for the rel tags

**Review by maxammann — CHANGES_REQUESTED** [review]
The title of the PR should be the same as the title on Jira. I fixed this!

Apart from the centering it looks good!

**Comment by maxammann** [comment]
You have a very goood Github name! Finally a name which I can remember!

Lookingn at you @Schedulaar, @klinzo and @Taggotty !

File: src/routes/sprungbrett/containers/SprungbrettOfferPage.js (hunk 1 above)
Comment (SteffiStuffel) [reply src/routes/sprungbrett/containers/SprungbrettOfferPage.js:42]: Fixed this.

**Review by maxammann — APPROVED** [review]

**Comment by maxammann** [comment]
The format for commits is actually "WEBAPP: Your commit message" But this is fine for now :)

**Pull request merged**